### PR TITLE
Use go-bindata built from vendor/

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
we should not rely on go-bindata from local environment

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: hack/generate-bindata.sh [hunk 1]
```diff
@@ -28,14 +28,12 @@ if [[ ! -d "${KUBE_ROOT}/pkg" ]]; then
 	exit 1
 fi
 
-# kube::golang::build_kube_toolchain installs the vendored go-bindata in
```
Comment (BenTheElder) [inline hack/generate-bindata.sh]: is this still the case / should that be cleaned up?

**Comment by BenTheElder** [comment]
/lgtm
/approve
thanks dims!

File: hack/generate-bindata.sh (hunk 1 above)
Comment (dims) [reply hack/generate-bindata.sh]: we don't have a build_kube_toolchain AFAICT - https://cs.k8s.io/?q=build_kube_toolchain&i=nope&files=&repos=kubernetes/kubernetes

**Comment by BenTheElder** [comment]
Excellent ...

**Comment by alejandrox1** [comment]
/milestone v1.18
Thank you Dims!

**Pull request merged**